### PR TITLE
SONARIAC-1458 Narrow the scope of YAML files considered by the spring-config sensor

### PR DESCRIPTION
**Pull request by petertrr**

### CODE REVIEW

File: iac-extensions/spring-config/build.gradle.kts [hunk 1]
```diff
@@ -11,6 +11,9 @@ dependencies {
 
     api(project(":iac-common"))
 
+    compileOnly(project(":iac-extensions:cloudformation"))
+    compileOnly(project(":iac-extensions:kubernetes"))
```
Comment (jonas-wielage-sonarsource) [inline iac-extensions/spring-config/build.gradle.kts:15]: Tbh I'm not that big of a fan of having this dependency, but there's no other way except duplicating and moving it to common.

**Review by jonas-wielage-sonarsource — COMMENTED** [review]
LGTM, please look at my comments.
Please fix the plugin test :)

File: its/ruling/src/integrationTest/resources/expected/spring-config/cloudformation-S6294.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
[the remark this answers is not in the document]
Comment (petertrr) [reply its/ruling/src/integrationTest/resources/expected/spring-config/cloudformation-S6294.json:1]: On the other hand, having other files lead us to notice this clash between sensors. So maybe it's okay to keep them.

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**